ping/src/test/java/com/example/ping/PingApplicationTests.java

```diff
@@ -1,13 +1,28 @@
 package com.example.ping;
 
 import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 
-@SpringBootTest
+import static org.assertj.core.api.Assertions.assertThat;
+
+@AutoConfigureMockMvc
+@WebMvcTest
 class PingApplicationTests {
+	@Autowired
+	private MockMvc mockMvc;
 
 	@Test
-	void contextLoads() {
+	void canPingService() throws Exception {
```

Comment:
This is a test for your current `/ping` endpoint. Having this test here means that you won't accidentally break it in the future when you rewrite large swathes of the code (which you're going to do).